quantii/src/lib.rs

```diff
@@ -70,9 +74,31 @@ impl ardaku::System for System {
 pub fn setup() -> ! {
     use ardaku::System;
 
-    System.write(b"\n\n=== ARDAKU STARTED ===\n");
+    System.write(b"Quantii OS v0.0.1");
+
+    System.write(b"\n=== ARDAKU STARTED ===\n");
+
+    let mut msg: String;
+    match ardaku::start(System, APP_EXE) {
+        Ok(_) => msg = "\n=== ARDAKU SUCCESSFULLY EXECUTED ===\n".to_owned(),
+        Err(e) => match e {
+            ArdakuError::InvalidWasm => {
+                msg = "Ardaku: Error: Invalid WASM file".to_owned()
+            }
+            ArdakuError::LinkerFailed => {
+                msg = "Ardaku: Error: Failed to link WASM file".to_owned()
+            }
+            ArdakuError::Crash(c) => {
+                msg = "Ardaku: Error: Crash:".to_owned();
+                msg += &c.to_string();
+            }
+            ArdakuError::MissingMemory => {
+                msg = "Ardaku: Error: Ran out of memory".to_owned()
+            }
+        },
+    }
```

Comment:
There's no need to use allocation here.  Since all of the messages are static `&str` types.